style.css

```diff
@@ -1524,6 +1520,26 @@ body {
 	background-position: center center;
 	background-repeat: no-repeat;
 	background-size: cover;
+	height: 165px;
+}
+
+.custom-header-image:before {
+	/* Permalink - use to edit and share this gradient: http://colorzilla.com/gradient-editor/#000000+0,000000+100&0+0,0.3+75 */
+	background: -moz-linear-gradient(top,  rgba(0,0,0,0) 0%, rgba(0,0,0,0.3) 75%, rgba(0,0,0,0.3) 100%); /* FF3.6-15 */
```

Comment:
1. Gradient should use `to top`, not `top` (A common mistake (matching outdated non-standard syntax) is to use just a side-or-corner without the preceding `to` ([src](http://stylelint.io/user-guide/rules/function-linear-gradient-no-nonstandard-direction/))

2. A single space after each comma is expected